src/Api/Models/User/User.cs

```diff
@@ -66,7 +66,7 @@ public bool IsStudent()
             return Role == BOL.Models.Role.Student;
         }
 
-        [ForeignKey("Id")]
-        public virtual List<Notification> NotificationsForUser { get; set; }
+        [ForeignKey("UserId")]
```

Comment:
If you don't know what this "[ForeignKey("UserId")]" effects, better leave it out, but UserId is even not the right name.